man/ocamlc.1

```diff
@@ -404,16 +405,16 @@ can help in writing an explicit interface (.mli file) for a file: just
 redirect the standard output of the compiler to a .mli file, and edit
 that file to remove all declarations of unexported names.
 .TP
-.B \-cmi-file \ filename
+.BI \-cmi-file " filename"
 Type-check the source implementation to be compiled against the
 specified interface file (by-passes the normal lookup for .mli and .cmi files).
 .TP
-.BI \-I \ directory
+.BI \-I " directory"
 Add the given directory to the list of directories searched for
 compiled interface files (.cmi), compiled object code files
 (.cmo), libraries (.cma), and C libraries specified with
-.BI \-cclib\ \-l xxx
-.RB .
+.BI \-cclib\ \-l xxx\c
+.R .
```

Comment:
This changes makes the `.` disappears in the man output.

Reply:
Ha! Multiple nits: the `.R` macro doesn't exist, one can't start a line with `.` and expect it to be printed as it's the control character, escaping it with `\.` doesn't work, but `\[char46]` does the trick.